presto-verifier/src/test/java/com/facebook/presto/verifier/TestPrestoVerifierIntegrationSmokeTest.java

```diff
@@ -80,6 +80,7 @@ public void setup()
     public void teardown()
     {
         closeQuietly(handle);
+        mySqlServer.close();
```

Comment:
We should also close the presto query runner. Also it is safer to do `closeQuietly`. e.g.: `closeQuietly(queryRunner, mySqlServer, handle);`